### PR TITLE
Try fixing output directory

### PR DESCRIPTION
**Pull request by canyon289**
Will also push a change that will fix categories as well

### CODE REVIEW

**Comment by canyon289** [comment]
Added tags and categories for everything

File: dev/pelicanconf.py [hunk 1]
```diff
@@ -18,12 +18,11 @@
 
 THEME = "custom"
 
-DELETE_OUTPUT_DIRECTORY = False
-
 SUMMARY_MAX_LENGTH = 0
 
 STATIC_PATHS = ['images', 'data', 'js', 'css']
 
+DELETE_OUTPUT_DIRECTORY = True
```
Comment (sarinac) [inline dev/pelicanconf.py:25]: Ohhh that's so handy!

Comment (sarinac) [reply dev/pelicanconf.py:25]: oh, just realized you originally had it in publishconf :O

**Review by sarinac — APPROVED** [review]
LGTM! Thanks for the amazing turnaround!

**Pull request merged**